service/config.py

```diff
@@ -2,6 +2,7 @@
 Global Configuration for Application
 """
 import os
+import logging
```

Comment:
While troubleshooting, I copied/pasted the contents of config.py in the BDD class lab into our config.py file. It doesn't look like anything broke?